devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -223,3 +240,45 @@ private getModelBindings(model) {
 	}
 	bindings
 }
+
+private Map parseAduroSmartButtonMessage(Map descMap){
+	def buttonState = "pushed"
+	def buttonNumber = 0
+	if (descMap.clusterInt == zigbee.ONOFF_CLUSTER) {
+		if (descMap.command == "01") {
+		    buttonNumber = 1
+		}else if (descMap.command == "00") {
+		    buttonNumber = 4
+		}else {
+		    return [:]
+		}
+	}else if (descMap.clusterInt == zigbee.LEVEL_CONTROL_CLUSTER) {
+		if (descMap.command == "02") {
+		    def data = descMap.data
+		    def d0 = data[0]
+		    if (d0 == "00") {
+			buttonNumber = 2
+		    }
+		    if (d0 == "01") {
```

Comment:
nit: this could be an `else if` instead

Reply:
Fixed issues

Reply:
@adurosmart this is still an `if` but should be changed to an `else if`

Reply:
Thank you review,Fixed issues.